java-lib/src/main/java/org/clink/linalg/SparseVectorJNA.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.clink.linalg;
+
+import org.apache.flink.ml.linalg.SparseVector;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.Structure.FieldOrder;
+
+/**
+ * Class that corresponds to struct SparseVectorJNA in C++. It is only used for JNA to transmit data
+ * between Java and C++.
+ */
+@FieldOrder({"n", "indices", "values", "length"})
+public class SparseVectorJNA extends Structure {
```

Comment:
The best practice in Java class naming might be to name this as `SparseVectorJna`. Could you double check?

Same for `ClinkJNA.java`.

Reply:
I can hardly tell, as I can find both usages of `Jna` and `JNA`. Still, I agree that `Jna` might better satisfies the camelCase naming. I'll make the change.